res/css/photon-components.css

```diff
@@ -121,3 +154,139 @@
 .photon-message-bar-error .photon-button:hover:active {
   background-color: var(--red-90);
 }
+
+/* Label */
+
+.photon-label,
+.photon-label-micro {
+  display: inline-flex;
+  color: var(--grey-90);
+}
+
+.photon-label {
+  padding: 3px 0;
+  font-size: 13px;
+}
+
+.photon-label-micro {
+  padding: 3px 0;
+  font-size: 11px;
+}
+
+/* Checkboxes */
+
+.photon-checkbox,
+.photon-checkbox-micro {
+  position: relative;
+  padding: 0;
+  border: 1px solid var(--grey-90-a30);
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--grey-90-a10);
+  border-radius: 2px;
+}
+
+.photon-checkbox {
+  width: 16px;
+  height: 16px;
+  margin: 0 4px;
+}
+
+.photon-checkbox-micro {
+  width: 13px;
+  height: 13px;
+  margin: 0 3px;
+}
+
+.photon-checkbox:hover,
+.photon-checkbox-micro:hover {
+  background-color: var(--grey-90-a20);
+}
+
+.photon-checkbox:active,
+.photon-checkbox-micro:active {
+  background-color: var(--grey-90-a30);
+}
+
+.photon-checkbox:checked,
+.photon-checkbox-micro:checked {
+  border: none;
+  background-color: var(--blue-60);
+}
+
+.photon-checkbox:checked {
+  background-image: url(../img/svg/checkbox-check-16.svg);
```

Comment:
I see there's a small delay to load it before displaying it. I've seen we can use [url-loader](https://webpack.js.org/loaders/url-loader/) instead of `file-loader` in the webpack config to automatically replace small assets by data URL. I think we should configure it, but this feels out of the scope of this PR. Do you think you could do it in a separate PR though?

Reply:
Ah actually we don't see the delay in the app itself, we see it only in the photon example page. Probably the service worker makes this problem disappear. What do you think?

Reply:
As long as it's not apparent in the main app I don't mind.